documentation/user_docs/docs/user_guide/Horace3_to_4.rst

```diff
@@ -0,0 +1,164 @@
+####################################
+Converting from Horace 3 to Horace 4
+####################################
+
+Horace 4 has been a major rewrite and refactoring effort on the code
+from Horace 3. This document outlines the breaking changes between
+versions as well as describing a fast way to convert existing scripts
+from Horace 3 to Horace 4.
+
+Projections
+-----------
+
+In Horace 4, projections have been expanded and reworked to be
+generalised for other shapes of cuts (e.g. spherical, cylindrical). As
+a result, the old-style projections, e.g.:
+
+.. code-block:: matlab
+
+   proj1 = struct('u', [1 0 0], 'v', [0 1 0]);
+   proj2 = projaxes([1 0 0], [0 1 0]);
+   proj3 = ortho_proj([1 0 0], [0 1 0]);
+
+   proj4.u = [1 0 0];
+   proj4.v = [0 1 0];
+
+have been deprecated and a warning will be issued if they are
+used. For old-style projections the conversion is as simple as
+replacing the ``struct``, ``projaxes`` or ``ortho_proj`` with ``line_proj``
+(c.f. :ref:`projections
+<manual/Cutting_data_of_interest_from_SQW_files_and_objects:Projection in more details>`).
+
+.. code-block:: matlab
+
+   proj1 = line_proj([1 0 0], [0 1 0]);
+   proj2 = line_proj([1 0 0], [0 1 0]);
+   proj3 = line_proj([1 0 0], [0 1 0]);
+   proj4 = line_proj([1 0 0], [0 1 0]);
+
+Symmetrisation
+--------------
+
+The function ``sqw.symmetrise_sqw`` has been significantly rewritten
+to generalise it and allow rotational symmetry reduction. As part of
+this rewrite, it now takes ``Symop`` s as arguments rather than a pair
+of vectors.
+
+To update e.g.:
+
+.. code-block:: matlab
+
+   symmetrised = win.symmetrise_sqw([1 0 0], [0 1 0], [1 1 1]);
+
+simply becomes:
+
+.. code-block:: matlab
+
+   symmetrised = win.symmetrise_sqw(SymopReflection([1 0 0], [0 1 0], [1 1 1]);
+
+The ``SymopReflection`` can also be constructed earlier and reused:
+
+.. code-block:: matlab
+
+   sym = SymopReflection([1 0 0], [0 1 0], [1 1 1]);
+   symmetrised = win.symmetrise_sqw(sym);
+
+
+Crystal Alignment
+-----------------
+
+Crystal alignment in Horace 4 has been changed such that alignment is
+a two-phase process. This is in order to prevent having to rewrite
+large datafiles repeatedly during the course of the realignment
+process.
+
+Multifit
+--------
+
+As of Horace 4, the deprecated legacy multifit syntax (i.e. all in one
+line):
+
+.. code-block:: matlab
+
+   [wfit, fitdata] = multifit_sqw(my_new_cut, @sr122_xsec, pars, pfree, pbind, 'list', 1);
+
+has been fully removed. This means that trying to use this syntax will
+result in an error. The modern syntax uses an object-based form which
+looks like:
+
+.. code-block:: matlab
+
+   kk = multifit(my_new_cut);
+   kk = kk.set_fun(@sr122_xsec);
+   kk = kk.set_pin(pars);
+   kk = kk.set_free(pfree);
+   kk = kk.set_bind(pbind);
+   kk = kk.set_options('listing', 1);
+   [wfit, fitdata] = kk.fit();
+
+while this would be a lot of effort to translate manually, thankfully,
```

Comment:
```suggestion
While this would be a lot of effort to translate manually, thankfully,
```
sorry to be pedantic but otherwise this looks like a qualification of the previous text